packages/quantic/cypress/integration/facets/numeric-facet/numeric-facet.cypress.ts

```diff
@@ -0,0 +1,163 @@
+import {configure} from '../../../page-objects/configurator';
+
+import {NumericFacetExpectations as Expect} from './numeric-facet-expectations';
+import {InterceptAliases, interceptSearch} from '../../../page-objects/search';
+import {field, NumericFacetActions as Actions} from './numeric-facet-actions';
+
+interface NumericFacetOptions {
+  field: string;
+  label: string;
+  numberOfValues: number;
+  sortCriteria: string;
+  rangeAlgorithm: string;
+  withInput: string;
+  isCollapsed: boolean;
+}
+
+describe('Numeric Facet Test Suite', () => {
+  const pageUrl = 's/quantic-numeric-facet';
+
+  const defaultField = 'ytlikecount';
+  const defaultLabel = 'Youtube Likes';
+  const defaultNumberOfValues = 8;
+
+  const defaultSettings = {
+    field: defaultField,
+    label: defaultLabel,
+    numberOfValues: defaultNumberOfValues,
+  };
+  const customWithInputSettings = {
+    field: defaultField,
+    label: defaultLabel,
+    numberOfValues: defaultNumberOfValues,
+    withInput: 'integer',
+  };
+
+  function visitNumericFacetPage(
+    options: Partial<NumericFacetOptions> = {},
+    waitForSearch = true
+  ) {
+    interceptSearch();
+    cy.visit(pageUrl);
+    configure(options);
+    if (waitForSearch) {
+      cy.wait(InterceptAliases.Search);
+    }
+  }
+
+  function loadFromUrlHash(
+    options: Partial<NumericFacetOptions> = {},
+    urlHash: string
+  ) {
+    interceptSearch();
+    cy.visit(`${pageUrl}#${urlHash}`);
+    configure(options);
+    cy.wait(InterceptAliases.Search);
+  }
+
+  describe('with default numeric facet', () => {
```

Comment:
The `describe`s under `with default numeric facet` could be grouped into a single test since they all use the same settings. It would avoid reloading the test page a couple times.

    it('should work as expected', () => {
      visitNumericFacetPage(defaultSettings);

      scope('on initial load', () => {
        Expect.logFacetLoad();
        ...
        Expect.numberOfIdleCheckboxValues(defaultNumberOfValues);
      });

      scope('when selecting a value', () => {
        Actions.checkValueAt(0);

        Expect.displayClearButton(true);
        ...
      });

      scope('when selecting many values', ...);
      scope('when clearing values', ...);
    });